jax/lax/lax.py

```diff
@@ -181,7 +181,7 @@ def digamma(x):
 
 def igamma(a, x):
   r"""Elementwise regularized incomplete gamma function."""
-  return igamma_p.bind(_brcast(a, x), _brcast(x, a))
```

Comment:
I think there are a few other primitives that need a similar change, e.g. `regularized_incomplete_beta`?

Could we fix them too?